### ISSUE TITLE
Issue with My Profile on committers.top/Brazil

### ISSUE THREAD

**Issue by FabianaCampanari** [isyuricunha/top-github-users#6]
Hey there,

I hope all is well on your end. I would like to bring to your attention an issue I have encountered with the recent update of committers.top, specifically related to the Brazil section.

Firstly, I notice that on your repository, my profile is listed in the second position, despite the recent update that caused my name to disappear from the committers.top/Brazil website. You can verify this by visiting the following link:

https://github.com/isyuricunha/top-github-users/blob/main/markdown/public_contributions/brazil.md

Secondly, from your repository, I discovered another repository on GitHub and once again my profile is listed in the second position. You can confirm this by visiting the following link:

[top-github-users/markdown/public_contributions/brazil.md at main · gayanvoice/top-github-users](https://github.com/gayanvoice/top-github-users/blob/main/markdown/public_contributions/brazil.md)

I believe it is important to inform you about this issue, as my profile is not being displayed even though I should still be in the list and in second position This is the first time this has happened and the latest update seems to made my profile disappear from the website."
Please, let me know if there's any other way I can help out. 

Keep up the good work !

Best
Fabiana 🚀 Campanari

**Comment by isyuricunha** [comment]
Hi Fabiana,

Thank you for reporting the bug, and I am already taking a look at it.

If you don't mind, I would like you to check the response I provided on Twitter/X (https://twitter.com/CampanariFabi/status/1718068472893059402).

Although git-top-user and committer-top projects are different, they share the same GitHub API.

One more thing I noticed is that apparently, only your profile disappeared from the ranking.

Below, I'll restate what I said on Twitter/X:

I believe there may have been a bug in the new GitHub ranking system; I still need to check on this.

Maybe there's one error in the request, like "brazil,+cityORstate."
Try one thing:
Remove your city/state and only use Brazil or Brasil.

I will check here.

Enter here: https://docs.github.com/en/graphql/overview/explorer, log in with GH, write the code/query below, run it, and show me the result.

Query:
```
query { 
  viewer { 
    login
    contributionsCollection {
      restrictedContributionsCount
      contributionCalendar {
        totalContributions
      }
    }
  }
}
```

**Comment by FabianaCampanari** [comment]
Thank you I already replyed your message on X and I let you know if Will work.
And yes, I am the only one who profile desappeared 😭😭😭

Changing Subjects,

I red some of your histories on your blog , and the last one about  your Friends and 
 feelings related to your Family made me feel touched....

What you wrote is it true or  just a historytelling ?

If it was for real I would like to say something that could be important for you ...if you dont mind.....

You are so sensitive and if what you wrote is real life, you should HEALL this situation , and that's not related about the others phisically is more about the "others'  living inside of you ....

I was surprised how sensitve your writting is...
And I got a emotive

One Day if you want to talk about you can Count on me.
I'am Psycholgist,  by the way...

Thank you, I Will rua the code 







Fabiana 🚀 Campanari 
👩‍💻 https://linktr.ee/fabianacampanari

**Comment by isyuricunha** [comment]
Fabiana,

As I informed you, it was just a GitHub indexing issue, and it has already been resolved. Both on Top Git User and on https://committers.top/, your profile has reappeared.

With that in mind, I will close the issue.
Take care.
Cheers,
Yuri Cunha.

**Comment by FabianaCampanari** [comment]
Hey Yuri,Just wanted to give you a quick shoutout and say thanks for being so on top of things with the GitHub indexing issue. It's awesome to see that my profile is back up on both Top Git User and https://committers.top/. You're killing it, man!You rock! 🚀FabianaOn Nov 1, 2023, at 11:30, Yuri Cunha ***@***.***> wrote:﻿
Closed #6 as completed.
